src/test_runner.js

```diff
@@ -285,7 +285,7 @@ TestRunner.prototype = {
     var sentry;
 
     var seleniumSessionId;
-    var stdout = "";
+    var stdout = clc.greenBright(logStamp()) + " Magellan child process start\n\n";
```

Comment:
Do you think having higher than second resolution would help here?  What about UTC time?